lib/internal/modules/cjs/helpers.js

```diff
@@ -99,18 +99,22 @@ function stripShebang(content) {
   return content;
 }
 
+const namespaceBuiltinLibs = [];
+
+if (getOptionValue('--experimental-worker')) {
+  namespaceBuiltinLibs.push('worker_threads');
+  namespaceBuiltinLibs.sort();
+}
+
 const builtinLibs = [
   'assert', 'async_hooks', 'buffer', 'child_process', 'cluster', 'crypto',
   'dgram', 'dns', 'domain', 'events', 'fs', 'http', 'http2', 'https', 'net',
   'os', 'path', 'perf_hooks', 'punycode', 'querystring', 'readline', 'repl',
   'stream', 'string_decoder', 'tls', 'trace_events', 'tty', 'url', 'util',
-  'v8', 'vm', 'zlib'
+  'v8', 'vm', 'zlib', ...namespaceBuiltinLibs
 ];
 
-if (getOptionValue('--experimental-worker')) {
-  builtinLibs.push('worker_threads');
-  builtinLibs.sort();
-}
+builtinLibs.sort();
```

Comment:
It seems like we have a lot of sorting here, we can move this below line 121 and remove the sorting inside the if clause to go with one sorting.